pyicloud/base.py

```diff
@@ -98,6 +121,14 @@ def authenticate(self):
             msg = 'Invalid email/password combination.'
             raise PyiCloudFailedLoginException(msg)
 
+        # Pull X-APPLE-WEB-KB cookie from cookies
+        NewWebKBCookie = next(({key:val} for key, val in req.cookies.items() if 'X-APPLE-WEB-KB' in key), None)
```

Comment:
Hey there @FiZiX -- could you elaborate on what you're trying to do here; I'm a little surprised to see `next` being used, and is the cookie name really not just `X-APPLE-WEB-KB`, or is it actually possible for there to be leading or trailing text on that?